packages/xdl/package.json

```diff
@@ -71,6 +71,7 @@
     "p-timeout": "2.0.1",
     "pacote": "9.2.3",
     "plist": "2.1.0",
+    "probe-image-size": "^4.0.0",
```

Comment:
@fson is it ok with you to add this library as an xdl dep?

Reply:
It should be fine, xdl even already depends on it indirectly, through `@expo/schemer`.